src/renderer/features/operation-details/governance-operation-details/governance-operation-details-model.tsx

```diff
@@ -0,0 +1,215 @@
+import { useUnit } from 'effector-react';
+import { useEffect, useState } from 'react';
+import { Trans } from 'react-i18next';
+
+import { type Address, TransactionType } from '@/shared/core';
+import { createFeature } from '@/shared/effector';
+import { useI18n } from '@/shared/i18n';
+import { toAccountId } from '@/shared/lib/utils';
+import { DetailRow, FootnoteText } from '@/shared/ui';
+import { Account } from '@/shared/ui-entities';
+import { Skeleton } from '@/shared/ui-kit';
+import { AssetBalance } from '@/entities/asset';
+import { TracksDetails, voteTransactionService } from '@/entities/governance';
+import { getTransactionFromMultisigTx } from '@/entities/multisig';
+import { networkModel } from '@/entities/network';
+import { operationDetailsUtils } from '@/entities/operations';
+import { isUndelegateTransaction } from '@/entities/transaction';
+import { multisigOperationsFeature } from '@/features/multisig-operations';
+
+import { GovernanceOperationTitle } from './components/GovernanceOperationTitle';
+
+export const governanceOperationDetailFeature = createFeature({
+  name: 'governance/operation-details',
+});
+
+governanceOperationDetailFeature.inject(multisigOperationsFeature.slots.operationDetails, {
+  render: ({ operation }) => {
+    const { t } = useI18n();
+    const transaction = getTransactionFromMultisigTx(operation);
+
+    const chains = useUnit(networkModel.$chains);
+    const apis = useUnit(networkModel.$apis);
+
+    const chain = chains[operation.chainId];
+    const api = apis[operation.chainId];
+
+    const defaultAsset = chain?.assets[0];
+
+    const [isUndelegationLoading, setIsUndelegationLoading] = useState(false);
+    const [undelegationVotes, setUndelegationVotes] = useState<string>();
+    const [undelegationTarget, setUndelegationTarget] = useState<Address>();
+
+    const result = [];
+
+    useEffect(() => {
+      if (isUndelegateTransaction(transaction)) {
+        setIsUndelegationLoading(true);
+      }
+
+      if (!api) return;
+
+      operationDetailsUtils.getUndelegationData(api, operation).then(({ votes, target }) => {
+        setUndelegationVotes(votes);
+        setUndelegationTarget(target);
+        setIsUndelegationLoading(false);
+      });
+    }, [api, operation]);
+
+    if (
+      transaction?.type &&
+      ![
+        TransactionType.UNLOCK,
+        TransactionType.VOTE,
+        TransactionType.REVOTE,
+        TransactionType.REMOVE_VOTE,
+        TransactionType.DELEGATE,
+        TransactionType.UNDELEGATE,
+        TransactionType.EDIT_DELEGATION,
+      ].includes(transaction.type)
+    ) {
+      return null;
+    }
+
+    const delegationTarget = operationDetailsUtils.getDelegationTarget(operation);
+    const delegationTracks = operationDetailsUtils.getDelegationTracks(operation);
+    const delegationVotes = operationDetailsUtils.getDelegationVotes(operation);
```

Comment:
I think we can keep it as is for now, but main idea is to fully decouple views **and** logic, so parsing of particular transaction info should be bounded to related context (domain or feature). In this case parsing of delegation details should be somewhere in governance services, not in operations utils.
Add `TODO` please!